bin/restic_backup.sh

```diff
@@ -101,3 +103,31 @@ wait $!
 #wait $!
 
 echo "Backup & cleaning is done."
+
+#
+# (optionally) Notify about backup summary stats.
+#
+# How to perform the notification is up to the user; the script only writes the info to the user-owned file in a fire
+# and forget fashion.
+#
+# One option to trigger desktop notifications on user-side is using a special FIFO file (a.k.a. pipe file), which will
+# work as a queue; plus, a user process to read from that queue and run a desktop notification command.
+#
+# TODO Clean/or rephrase the example comments below
+# In this case I'm running a user process that reads from a special pipe file and sends a desktop notification using
+# `notify-send`.
+# 
+# See: https://github.com/gerardbosch/dotfiles-linux/blob/main/home/.config/autostart/notification-queue.desktop and
+#      https://github.com/gerardbosch/dotfiles-linux/blob/main/home/bin/notification-queue-start-processing
+#
+if [ "$RESTIC_NOTIFY_BACKUP_STATS" = true ]; then
+	if [ -w "$RESTIC_BACKUP_NOTIFICATION_FILE" ]; then
+		added=$(grep -i 'Added to the repo:' <<< "$backup_output" | sed -E 's/.*dded to the repo: (.*)/\1/')
```

Comment:
At least my editor (vim) becomes very confused with the here string syntax. I think it would be more readable as well using simple echo (as I think most people don't know abut thee here string feature).

```bash
added=$(echo "$backup_output" | grep -i 'Added to the repo:' | sed -E 's/.*dded to the repo: (.*)/\1/')
```

Is there some benefit of using the here string here? I assume the intention is to avoid "useless-echo", but it comes at a cost of readability

Reply:
Ups! I'm using VIM and didn't notice anything strange with the `<<<`, hehe. But I've updated it for the `echo ... | ` syntax :+1: :+1: 

I used the `<<<` as it looked shorter and probably saved one process (the `echo`), so made me quite sense. But after looking at it, I found that the "here string" is implemented with temporary files :-S https://stackoverflow.com/a/41899209/6108874